modules/solid_mechanics/src/vectorpostprocessors/AverageSectionValueSampler.C

```diff
@@ -40,12 +40,19 @@ AverageSectionValueSampler::validParams()
   params.addDeprecatedParam<std::vector<Real>>(
       "lengths",
       {},
-      "Positions along axis of from reference_point at which to compute average values.",
+      "Positions along axis from reference_point at which to compute average values.",
       "Use the 'positions' parameter instead");
   params.addParam<std::vector<Real>>(
       "positions",
       {},
-      "Positions along axis of from reference_point at which to compute average values.");
+      "Positions along axis from reference_point at which to compute average values.");
+  params.addParam<RealVectorValue>(
+      "symmetry_plane",
+      RealVectorValue(0, 0, 0),
+      "Vector normal to a symmetry plane, used if the section has a symmetry plane through it. "
```

Comment:
```suggestion
      "Vector normal to a symmetry plane, used if the section has a symmetry plane (including the origin) through it. "
```

Reply:
I don't know what you mean by this. It's just a vector normal to the plane. No information about the origin of the plane is needed.

Reply:
the plane goes through the origin though? It does not have an equation like x = 1, it s necessarily x + 2y + 3z = 0 for example (goes through the origin)

Reply:
ok no it s not limited to that